tests/dragonfly/tls_replication_test.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import asyncio
+import redis
+from redis import asyncio as aioredis
+from . import DflyInstanceFactory, dfly_args
+import logging
+import time
+from dragonfly.replication_test import check_all_replicas_finished
+
+BASE_PORT = 1111
+ADMIN_PORT = 1211
+
+"""
+Test replication with tls and non-tls options
+"""
+
+# 1. Number of master threads
+# 2. Number of threads for each replica
+replication_cases = [(8, 8)]
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("t_master, t_replica", replication_cases)
+@dfly_args({"dbfilename": "test-tls-replication-{{timestamp}}", "admin_nopass" : True})
```

Comment:
I believe you can drop the dbfilename param
why is the admin_nopass needed?

Reply:
to make our life easier so I don't have to setup a password

Reply:
why password is required if requirepass flag is not set?

Reply:
because from my understanding you still need to provide an auth token according to https://github.com/dragonflydb/dragonfly/issues/1181

Reply:
I quote:

>However for auth protected setups we still need to provide an auth token for clients that connect via the admin port.